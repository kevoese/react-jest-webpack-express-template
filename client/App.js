import React from 'react';
import ReactDOM from 'react-dom';
import Main from './Main';
import './app.scss';

ReactDOM.render(
  <Main />,
  document.getElementById('root') || document.createElement('div')
);

if (module.hot) module.hot.accept();
