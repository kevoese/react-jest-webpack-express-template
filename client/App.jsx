import React from 'react';
import ReactDOM from 'react-dom';
import Main from './Main';
import './styles/index.scss';
import './styles/tailwind.css';
import './index.css';

ReactDOM.render(
  <Main />,
  document.getElementById('root') || document.createElement('div')
);

if (module.hot) module.hot.accept();
