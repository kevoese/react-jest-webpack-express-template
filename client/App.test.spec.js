import App from "./App";

describe("Index file", () => {
  it("renders the app file without crashing", () => {
    expect(
      JSON.stringify(
        Object.assign({}, App, { _reactInternalInstance: "censored" }),
      ),
    ).toMatchSnapshot();
  });
});