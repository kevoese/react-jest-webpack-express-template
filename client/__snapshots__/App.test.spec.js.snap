// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index file renders the app file without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
